shiny/test/playwright.py

```diff
@@ -0,0 +1,17 @@
+from playwright.sync_api import BrowserContext as BrowserContext
+from playwright.sync_api import FilePayload as FilePayload
+from playwright.sync_api import FloatRect as FloatRect
+from playwright.sync_api import Locator as Locator
+from playwright.sync_api import Page as Page
+from playwright.sync_api import Position as Position
+from playwright.sync_api import expect as expect
+
```

Comment:
Please rename this file to `_playwright.py` to hide it.
(This will cause many import updates)